pkg/manager/rest.go

```diff
@@ -21,9 +22,12 @@ func List[T runtime.Object](ctx context.Context, client *rest.RESTClient, resour
 		Into(obj)
 
 	if err != nil {
+		log.Tracef("manager.List -  resource [%s] - namespace [%s] - error [%s]", resource, namespace, err)
```

Comment:
What would be the best practice to make this trace using a variable number of parameters in Go? @enrichman

Reply:
See `Logger.with()` https://go.dev/blog/slog